### PR TITLE
Build docker image against node 16

### PR DESCRIPTION
**Pull request by Half-Shot**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ environments.
 
 Hosting this bridge requires you to have a Matrix homeserver. In order to
 connect a Slack Workspace to your bridge, you will need permission to add bots
-to it. You will also need Node.JS 10+ or Docker on your system.
+to it. You will also need Node.JS 12+ or Docker on your system.
```
Comment (Half-Shot) [inline README.md:19]: We've always expected Node 12+, but failed to document it.

File: Dockerfile [hunk 2]
```diff
@@ -1,21 +1,21 @@
-FROM node:14-alpine AS BUILD
+FROM node:16-alpine AS BUILD
 
 # git is needed to install Half-Shot/slackdown
 RUN apk add git
 WORKDIR /src
 
 COPY package.json package-lock.json /src/
-RUN npm install
+RUN npm install --ignore-scripts
```
Comment (tadzik) [inline Dockerfile:8]: Isn't this a good time to switch it to `npm ci`, so that we actually respect what's in package-lock?

Comment (Half-Shot) [reply Dockerfile:8]: This is an excellent time for that, how could I forget.

**Comment by Half-Shot** [comment]
Assuming otherwise happiness.

**Pull request merged**